manifest.yml

```diff
@@ -9,7 +9,7 @@ applications:
     NEW_RELIC_APP_NAME: Pulse | Prod
     NEW_RELIC_ENV: production
 - name: pulse-staging
-  memory: 128M
+  memory: 512M
```

Comment:
The staging site had an out of memory error, so I bumped it up to match the production site.